### ISSUE TITLE
Site is down - domain expired, I think?

### ISSUE THREAD

**Issue by ExplodingCabbage** [guillermooo/sublime-undocs#270]
http://docs.sublimetext.info is down. I get an ERR_NAME_NOT_RESOLVED in Chrome, and 
I think https://mxtoolbox.com/SuperTool.aspx?action=a%3adocs.sublimetext.info&run=toolpage# confirms that the domain has no DNS records. Looks like somebody needs to reregister the domain!

---

Edit by @FichteFoll: Replacement is up at https://docs.sublimetext.io/.

**Comment by themilkman** [comment]
Until this is resolved, everyone could use https://sublime-text-unofficial-documentation.readthedocs.io/

**Comment by FichteFoll** [comment]
As an update, I tried reaching @guillermooo but so far he hasn't responded. I do not have access to the domain records, nor does the domain seem to be up for sale at the moment.

**Comment by FichteFoll** [comment]
I still haven't heard a word from @guillermooo, unfortunately.

**Comment by ExplodingCabbage** [comment]
Doesn't look like he's been active on GitHub since October. How'd you try to reach out to him? Looks to me like in addition to the email address listed in his GitHub profile he has this Twitter account (I assume it's his!) from which he's Tweeted in December: https://twitter.com/guillermooo

In case he's not currently logging into GitHub regularly nor checking his email account, maybe Tweeting at him would succeed at getting his attention?

**Comment by ExplodingCabbage** [comment]
I've gone ahead and done so: https://twitter.com/XplodingCabbage/status/1209112938042318848

**Comment by FichteFoll** [comment]
Thanks for the research. I tried contacting him via email, which has worked in the past but doesn't appear to be working right now.

@guillermooo, if you happen to read this via @ExplodingCabbage's tweet, please check your mail.

**Comment by themilkman** [comment]
Hi @guillermooo, is there any chance that you'd pass over the domain to someone else in order to make it working again? Thanks in advance for responding!

**Comment by FichteFoll** [comment]
With the help of @TheSecEng, I rehosted the site on https://docs.sublimetext.io/ by going back in the commit history of the last commit before the more restrictive no-derivatives license and applying everything I could on top of it. I also updated the site's landing page to reflect this change.

Please use this site in the future. ~~I will also add a commit describing the current situation in the README to this repo later.~~ Edit: done

**Comment by ExplodingCabbage** [comment]
@FichteFoll do you now expect the domain move to be permanent? If so, this weekend I'll edit the [86 SO posts](https://stackoverflow.com/search?q=docs.sublimetext.info) that point to the old domain to point to the new one.

**Comment by FichteFoll** [comment]
Yes, I consider the old domain to be dead. Please also note that with the restructuring of the community docs project all paths have changed as well. We used that opportunity to clean up at least some of the weird structure.

**Comment by themilkman** [comment]
Hi @guillermooo! I see that the website got reachable again recently :)

Is the domain still under your control? Are you willing to cooperate with the  [sublimetext.io](https://docs.sublimetext.io/) fork?

Cheers and thanks for any response!
